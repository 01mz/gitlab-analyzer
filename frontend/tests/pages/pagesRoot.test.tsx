import React from 'react';
import Index from '../../pages';
import {mount, ReactWrapper} from "enzyme";

describe("Pages Root", () =>{
    const mockEnqueue = jest.spyOn(require('notistack'), "useSnackbar");
    let enqueueSnackbar = jest.fn();
    const pushRouter = jest.spyOn(require('next/router'), 'useRouter');
    let push = jest.fn();
    let rend:ReactWrapper;

    beforeAll(async () =>{
        mockEnqueue.mockImplementation(() => {return {enqueueSnackbar}});
        pushRouter.mockImplementation(() => {return {push}})
        rend = mount(<Index/>);
        await Promise.resolve();
    })

    it("Snapshot Index", () => {
        expect(rend).toMatchSnapshot();

    })


})