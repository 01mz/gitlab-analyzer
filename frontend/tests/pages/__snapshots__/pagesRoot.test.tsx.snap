// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Root Snapshot Index 1`] = `<Home />`;
